doc/api/worker_threads.md

```diff
@@ -51,7 +51,8 @@ overhead of creating Workers would likely exceed their benefit.
 
 When implementing a worker pool, use the [`AsyncResource`][] API to inform
 diagnostic tools (e.g. in order to provide asynchronous stack traces) about the
-correlation between tasks and their outcomes.
+correlation between tasks and their outcomes. See
+[here][async-resource-worker-pool] for an example implementation.
```

Comment:
I know it's more verbose, but what do you think about being specific about where the link is going, like this?:

```suggestion
["Using `AsyncResource` for a `Worker` thread pool"][async-resource-worker-pool]
in the `async_hooks` documentation for an example implementation.
```

As I understand it, [using _here_ for link text is an accessibility issue](https://www.washington.edu/accessibility/links/).